lib/Demangling/Demangler.cpp

```diff
@@ -2829,6 +2829,8 @@ NodePointer Demangler::demangleFunctionEntity() {
       Args = TypeAndMaybePrivateName; Kind = Node::Kind::Allocator; break;
     case 'c':
       Args = TypeAndMaybePrivateName; Kind = Node::Kind::Constructor; break;
+    case 'F':
```

Comment:
TODO: Add mangling/demangling tests.
Currently, there are no SIL tests for `call` declarations.
I informally checked that demangling works via `-emit-silgen`.
The old remangler might not work - haven't look into how to invoke/test it yet.